lib/cli/src/Cardano/CLI.hs

```diff
@@ -729,6 +730,12 @@ cmdWalletGetUtxoStatistics mkClient =
 data TransactionFeatures = NoShelleyFeatures | ShelleyFeatures
     deriving (Show, Eq)
 
+-- | which json schema to use for output, True is simple
+metadataSchemaOption :: Parser Bool
```

Comment:
May I suggest you using a dedicated sum type for this case (e.g. like the one above) to avoid [boolean blindness](https://existentialtype.wordpress.com/2011/03/15/boolean-blindness/) ?
JFYI there is such package https://hackage.haskell.org/package/choice-0.2.2/docs/Data-Choice.html (I am not suggesting it here)

Reply:
Done